Resources/Prototypes/Entities/Objects/Specific/Medical/surgery.yml

```diff
@@ -203,3 +215,7 @@
         Brute: 20
     hitSound:
       path: /Audio/Items/drill_hit.ogg
+  - type: Tool
+    qualities:
+      - Sawing
+    speed: 0.5
```

Comment:
these speed values are all backwards. the improvised one is currently the best and advanced is the worst

Reply:
I am going to rip out my fingernails with pliers